manifests/HandyOrg/HandyWinGet-GUI/1.3.0.1.yaml

```diff
@@ -0,0 +1,16 @@
+Id: HandyOrg.HandyWinget-GUI
+Version: 1.3.0.1
+Name: HandyWinget-GUI
+Publisher: HandyOrg
+License: MIT
+LicenseUrl: https://github.com/HandyOrg/HandyWinGet-GUI/blob/master/LICENSE
+AppMoniker: handywinget
+Tags: winget, handy, handycontrol, handywinget, gui, package, packages, package manager 
+Description: GUI for installing apps through WinGet and Creating Yaml file
+Homepage: https://github.com/HandyOrg/HandyWinGet-GUI
+InstallerType: MSIX
+Installers:
+  - Arch: x86
+    Url: https://github.com/HandyOrg/HandyWinGet-GUI/releases/download/v1.3.0.1/HandyWinGet_x86.msix
```

Comment:
The new URL should be:

https://github.com/HandyOrg/HandyWinGet-GUI/releases/download/v1.3.0.1/setup.exe

Also change the Installer type to EXE

Reply:
This setup also has a signature and smart screen problem I'm looking for a way to sign setup without any problems